main.ts

```diff
@@ -0,0 +1,129 @@
+import "https://deno.land/std@0.210.0/dotenv/load.ts";
+import { serveDir } from "https://deno.land/std@0.207.0/http/file_server.ts";
+import { Eta } from "https://deno.land/x/eta@v3.1.0/src/index.ts";
+import { fetchAlaCarte, fetchMenu, Menu } from "./dagsen.ts";
+
+const {
```

Comment:
Could add a .env example or similar with examples for how these variables should look